bowtie2_runner.py

```diff
@@ -0,0 +1,2 @@
+from Bio import Entrez
+import subprocess
```

Comment:
This is a good start! Python is a bit weird with how it deals with external programs, but mostly I think you'll want to use `Popen`. [See here](https://github.com/Klepac-Ceraj-Lab/bifido_hmo/blob/master/lib/entrez_blast_nucl.py#L24-L27) in Lauren's repo as an example